buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -202,6 +202,11 @@ def build(OPENJDK_REPO, OPENJDK_BRANCH, SHAS, OPENJ9_REPO, OPENJ9_BRANCH, OMR_RE
     if (!BUILD_LIST) {
         BUILD_LIST = ''
     }
+    BUILD_IDENTIFIER = params.BUILD_IDENTIFIER
```

Comment:
variableFile.set_job_variables('wrapper') sets BUILD_IDENTIFIER. It this required?

Reply:
I think its fine. It will add it to the build description. I think having it on all build types is more simple than adding it for all but 1.